source/System.h

```diff
@@ -196,12 +199,18 @@ class System {
 	const Government *government = nullptr;
 	std::string music;
 
-	// Hyperspace links to other systems.
+	// All possible hyperspace links to other systems.
 	std::set<const System *> links;
+	// Only those hyperspace links to other systems that are accessible.
+	std::set<const System *> accessibleLinks;
+	// Other systems that can be accessed from this system via a jump drive at various jump ranges.
 	std::map<double, std::set<const System *>> neighbors;
 
-	// Defines whether this system can be seen when not linked.
+	// Defines whether this system can be seen when not linked. A hidden system will
+	// not appear when in view range, only when linked to a visited system.
```

Comment:
```suggestion
	// not appear when in view range, except when linked to a visited system.
```
is this what you meant?